backend/app/views/backend/shared/_index_items_search_and_filter.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="btn js-index-searcher-button">
+  <svg class="icon">
+    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-search"></use>
+  </svg>
+</div>
+<div class="btn">
+  <svg class="icon">
+    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-sort"></use>
+  </svg>
+</div>
+<div class="searcher js-index-searcher" <% if @search.present? %>style="display: block;"<% end %>>
```

Comment:
I don't see a way to disable the search, once we start searching. Clearing the field and pressing enter doesn't do anything. And the search input is always visible.